docs/installation.md

```diff
@@ -40,6 +48,26 @@ Note that by default, Jupyter Notebook will not be able to open a browser window
 WSL2 [does support](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gpu-compute) GPU compute, at least with nvidia cards, but setting it up [takes some work](https://developer.nvidia.com/cuda/wsl).
 :::
 
+
+(installation/m1)=
+
+#### macOS with M1 chips
+
+Apple Silicon (M1 mac, `osx-arm64`) is supported, but only through Rosetta because (as of August 2022) some upstream dependencies are not yet built on it.
+To configure `conda` to [use Rosetta](https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon) when creating an environment (or modifying one, such as installing new packages into it), prepend the command with `CONDA_SUBDIR=osx-64`:
```

Comment:
```suggestion
The Toolkit supports Apple Silicon (M1, M2, etc.), but only through [Rosetta] because (as of August 2022) some upstream dependencies are not yet built for the `osx-arm64` architecture. 
Conda can be configured to [use Rosetta] with the `CONDA_SUBDIR=osx-64` environment variable or the `subdir` Conda config variable. 
We recommend using this on a per-environment basis so that it persists across updates and new installs, but does not affect existing setups:
```

I think we should give a clear recommendation so that people can just get something working without too much thought. And I need to break up this suggestion so GitHub  doesn't think the suggestion code block ends early...